tests/boot/boot_from_pxe.pm

```diff
@@ -222,7 +222,9 @@ sub run {
         save_screenshot;
         # We have textmode installation via ssh and the default vnc installation so far
         if (check_var('VIDEOMODE', 'text') || check_var('VIDEOMODE', 'ssh-x')) {
-            $is_nvdimm && check_var('VIDEOMODE', 'text') ? type_string_very_slow('DISPLAY= ') : type_string_slow('DISPLAY= ');
+            #julie's workaround to trigger an arm test
+            #will remove before PR merge
+#            $is_nvdimm && check_var('VIDEOMODE', 'text') ? type_string_very_slow('DISPLAY= ') : type_string_slow('DISPLAY= ');
```

Comment:
reminder for removal before merge

Reply:
ok. thanks. The top commit is the workaround, I'll remove that commit after validation runs.